build_packages.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+git clean -fdx
```

Comment:
I'm also in favor of removing it. I suspect it was included in the original `build_packages.py` since this had to be run locally, but since we are moving to CI, where the checkout will always be clean, there is no value for release builds, and it's fairly anti-social if you're just testing locally.